docs/framework/security/building-my-first-claims-aware-aspnet-web-app.md

```diff
@@ -27,11 +27,11 @@ author: "BrucePerlerMS"
   
  STS is a service that issues tokens upon successful authentication. Microsoft offers two industry standard STS’s:  
   
-- [Active Directory Federation Services (AD FS) 2.0](https://go.microsoft.com/fwlink/?LinkID=247516)
+- [Active Directory Federation Services (AD FS) 2.0](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/dd727958(v=ws.10))
```

Comment:
I would roll back changes under the security folder since I'm actually trying to archive this content in PR #14590